packages/playground/interactive-block-playground/package.json

```
@@ -0,0 +1,15 @@
+{
+	"name": "interactive-block-playground",
+	"version": "1.0.0",
+	"description": "",
+	"main": "index.js",
+	"scripts": {
+		"start": "concurrently 'npm:compile-ts' 'npm:build-zips' 'npm:start-server'",
+		"compile-ts": "tsc -w -p tsconfig.json",
+		"build-zips": "node build-zips.js",
+		"start-server": "http-server"
+	},
+	"author": "The WordPress Contributors",
+	"license": "ISC",
+	"type": "module"
```

Let's avoid accidental publishing to npm:

```suggestion
	"type": "module",
	"private": "true"
```